ruby_on_rails/linting_and_automatic_check.md

```diff
@@ -10,6 +10,7 @@ Check out the `bin/fastcheck` [fastcheck](../templates/bin/fastcheck) for the fi
 ```ruby
 group :development, :test do
   gem 'rubocop', require: false
+  gem 'rubocop-faker', require: false
```

Comment:
as stated in the README, it's basically a helper to upgrade from Faker 1 to Faker 2: "RuboCop Faker is a tool for converting your Faker's methods to the latest Faker argument style with static code analysis."
For this reason I wouldn't make it part of the standard setup.

Reply:
Agree! I missed it. I'll close this PR